app/Repositories/Interfaces/BaseRepositoryInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Repositories\Interfaces;
+
+interface BaseRepositoryInterface
+{
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Interfaces/BaseRepositoryInterface.php</br>**Line**: 8</br>Expected 1 newline at end of file; 0 found